### PR TITLE
Require "admin" tag to create new users or modify existing users

### PR DESCRIPTION
**Pull request by grantshurtz**
Requires a user to be tagged as an admin to create or modify other user accounts.

### CODE REVIEW

File: src/routes/user.js [hunk 1]
```diff
@@ -59,11 +59,17 @@ router.post("/v1/user", async function (req, res) {
       }
 
       // First user has to have a password for security purposes
-      if (!password) {
+      // First user also has to be an admin, otherwise no other users can be created
+      if (!password || !tags.includes("admin")) {
```
Comment (MikeVerdicchio) [inline src/routes/user.js:63]: Should this be an `&&`? The first user needs to be an admin AND have a password?

Comment (grantshurtz) [reply src/routes/user.js:63]: That's the intent, both admin tag and password required - 

I think it's the same either way (i.e. demorgan's law, `!(a AND b) = (!a OR !b)` )

Happy to change it to the other version if you think that's more readable

Comment (MikeVerdicchio) [reply src/routes/user.js:63]: Oh haha you're totally right! Man I haven't thought about Demorgan's Law in a while 😆 Good to go!

**Review by MikeVerdicchio — APPROVED** [review]
LGTM

**Pull request merged**